2022/01.md

```diff
@@ -94,8 +94,8 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 2     | 60m     | Symbols as WeakMap Keys updates and discuss the [usage of registered and well-known keys](https://github.com/tc39/proposal-symbols-as-weakmap-keys/issues/21) ([Slides](https://docs.google.com/presentation/d/1-Hp4Qd0mmdufYCwI9fTdCxgt4xEHqE1mPfqRxzuzMvc/edit#slide=id.p)) | Robin Ricard, Leo Balter |
     |   | 1     | 30m     | [Intl.Segmenter v2](https://github.com/tc39-transfer/proposal-intl-segmenter-v2) for Stage 2 ([Slides](https://docs.google.com/presentation/d/1BJl99uYveimKrMw605KyaZ0qLthIhNaqONDPwdYH53A), [Spec Text](https://tc39-transfer.github.io/proposal-intl-segmenter-v2/)) | Frank Yung-Fong Tang |
     |   | 1     | 30m     | [Class brand check](https://github.com/tc39/proposal-class-brand-check) for Stage 2 (Slides TBD, [Spec Text](https://tc39.es/proposal-class-brand-check/)) | HE Shi-Jun |
+    |   | 1     | 45m     | [enum](https://github.com/Jack-Works/proposal-enum/) for stage 1 ([Slides](https://docs.google.com/presentation/d/14WtGmdWjEYXIXZVWJWpERF98D90_BytceAu7b7DKr5Q/edit?usp=sharing)) | Jack Works |
```

Comment:
The column is for the current stage, which is 0, not 1.

Reply:
Oh! Sorry!